plugins/inputs/filecount/README.md

```diff
@@ -19,31 +31,38 @@ Counts files in directories that match certain criteria.
   ## Only count regular files. Defaults to true.
   regular_only = true
 
-  ## Only count files that are at least this size in bytes. If size is
-  ## a negative number, only count files that are smaller than the
-  ## absolute value of size. Defaults to 0.
-  size = 0
+  ## Only count files that are at least this size. If size is a 
+  ## negative number, only count files that are smaller than the
+  ## absolute value of size. Defaults to "0B".
+  ## For available units, see :
+  ## https://godoc.org/github.com/alecthomas/units#pkg-constants
+  size = "0B"
 
   ## Only count files that have not been touched for at least this
   ## duration. If mtime is negative, only count files that have been
   ## touched in this duration. Defaults to "0s".
   mtime = "0s"
+
+  ## Only output directories whose sub elements weighs more than this
+  ## size. Defaults to "0B".
+  recursive_print_size = "0B"
```

Comment:
This would be better done as a processor.  We kind of toyed around with this in a [threshold processor](https://github.com/influxdata/telegraf/pull/4592/files#diff-8b33a517a228dce95ce6cd88080d8447), but we need to finish spec'ing out the processor.